crates/oxc_linter/src/rules/unicorn/prefer_negative_index.rs

```diff
@@ -300,7 +300,7 @@ fn test() {
         "[1, 2, 3].slice([1, 2, 3].length - 1)",
         "foo[bar++].slice(foo[bar++].length - 1)",
         "foo[a + b].slice(foo[a + b].length - 1)",
-        "foo[`${bar}`].slice(foo[`${bar}`].length - 1)",
+        // "foo[`${bar}`].slice(foo[`${bar}`].length - 1)",
```

Comment:
I think this should be moved to fail case.
If my understanding is incorrect, please tell me🙇‍♂️

If correct, I will move this case to fail.